src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -147,12 +239,14 @@ protected virtual Task Stop()
     }
 
 #if Version5
-    Task CreateOrPurgeQueues()
+    async Task CreateOrPurgeQueues()
     {
         var configuration = CreateConfiguration();
         if (IsPurgingSupported) configuration.PurgeOnStartup(true);
```

Comment:
@dennis Some transports do support purging (MSMQ, SQL), some support async purging (ASQ) some don't at all. This makes sure that the incoming queue is empty no matter what the support for purging is.